packages/node_modules/pouchdb-adapter-idb/src/bulkDocs.js

```diff
@@ -168,17 +169,28 @@ function idbBulkDocs(dbOpts, req, opts, api, idb, callback) {
     callback(null, results);
   }
 
-  function verifyAttachment(digest, callback) {
-
+  function verifyAttachment(att, callback) {
+    var digest = att.digest;
     var req = attachStore.get(digest);
     req.onsuccess = function (e) {
       if (!e.target.result) {
-        var err = createError(MISSING_STUB,
-          'unknown stub attachment with digest ' +
-          digest);
-        err.status = 412;
-        callback(err);
+        console.log('ATT HERE', att);
+        if (att.unfetched) {
```

Comment:
I think this should be moved to before verify attachment is even called since we already know to not expect it with an `if (att.unfetched` check

Reply:
I originally had it there. But this for the case we replicate back to the original source db. Then we can remove the unfetched flag since the attachment exists.